donate/settings/staging.py

```diff
@@ -0,0 +1,31 @@
+from .environment import env
+from .base import Base
+from .secure import Secure
+from .database import Database
+from .redis import Redis
+from .s3 import S3
+from .braintree import Braintree
+from .sentry import Sentry
+from .thunderbird import ThunderbirdOverrides
+
+
+class Staging(Base, Secure, Database, Redis, S3, Braintree, Sentry):
+    DEBUG = False
+    SECRET_KEY = env('DJANGO_SECRET_KEY')
+    BASKET_API_ROOT_URL = env('BASKET_API_ROOT_URL')
+    BASKET_SQS_QUEUE_URL = env('BASKET_SQS_QUEUE_URL')
+
+    @classmethod
+    def pre_setup(cls):
+        cls.LOGGING['loggers']['rq.worker']['handlers'] = ['info']
+        cls.LOGGING['loggers']['rq.worker']['level'] = 'INFO'
+
+
+class ThunderbirdStaging(Staging, ThunderbirdOverrides):
```

Comment:
Is there a way to make the ThunderbirdOverrides automatically hook into the `INSTALLED_APPS` and `TEMPLATES` variables? Or does it not have access to "the object that's been created so far" in a multiple inheritance chain?

Reply:
I initially tried that, but django-configurations doesn't seem to work that way. The solution I ended up with is the first combination of configurations that actually did what I was trying to do.